autovmaf_generate_excel.py

```diff
@@ -11,14 +14,16 @@
 from openpyxl.formatting.rule import CellIsRule
 
 
+logger = logging.getLogger("autovmaf-excel")
```

Comment:
This is the standard pythonic way to handle logging with support for different loglevels. The logger name is not used in practice, but can be with a custom Log formatter (which is also how you add color support). I didn't feel the need to go that far though.